content/posts/scientific-python/translations/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Setting up and managing translations for Scientific Python projects"
+date: 2024-08-08
+draft: false
+tags:
+  [
+    "Scientific-Python",
+    "translations",
+    "inclusion",
+    "CZI",
+  ]
+description: "Setting up and managing translations for Scientific Python projects."
+displayInList: true
+author: ["Melissa Mendonça", "Albert Steppi", "Tania Allard", "Ralf Gommers", "Stéfan van der Walt", "Jarrod Millman"]
+---
+
+In November 2022, [the Chan Zuckerberg Initiative (CZI) awarded the Scientific Python project with a grant to improve communications infrastructure and accessibility](https://blog.scientific-python.org/scientific-python/2022-czi-grant/). This proposal involves several key areas to improve sustainability, inclusivity and accessibility of the Scientific Python ecosystem. One of these areas is making software documentation and user interfaces available in multiple languages. We are happy to announce that we are now able to develop and publish translations of multiple project websites.
+
+In this blog post, we will discuss how we set up and plan to manage translations for Scientific Python projects in collaboration with Quansight Labs, and how you can participate in the translation and localization effort.
+
+## Why translations are important
+
+Accessibility and inclusion are important aspects of software development, and it is important that we focus on them when considering the global reach of our open-source projects and communities. By providing translations of our websites, documentation and user interfaces in multiple languages, we can reach a wider audience and make our projects more inclusive. This is especially important for scientific software projects, which are used by researchers and scientists from around the world with a wide range of language proficiencies and backgrounds.
```

Comment:
```suggestion
Accessibility and inclusion are important aspects of building a healthy community around open source software development. By providing translations of our websites, documentation and user interfaces in multiple languages, we reach a wider global audience, thereby making our projects more inclusive and increasing the diversity of contributions and ideas. This is especially important for scientific software projects, used by researchers and scientists from around the world with a wide range of language proficiencies and backgrounds.
```

This is mainly general edits, with a minor added emphasis on what we gain through diversity: healthier community, diversity of ideas.